pkg/utils/status.go

```diff
@@ -341,50 +364,67 @@ func calculateState(ctx context.Context, instance *csmv1.ContainerStorageModule,
 	controllerStatus := newStatus.ControllerStatus
 
 	newStatus.State = constants.Failed
-	log.Infof("deployment controllerReplicas [%s]", controllerReplicas)
-	log.Infof("deployment controllerStatus.Available [%s]", controllerStatus.Available)
+	//log.Infof("deployment controllerReplicas [%s]", controllerReplicas)
+	//log.Infof("deployment controllerStatus.Available [%s]", controllerStatus.Available)
 
-	log.Infof("daemonset expected [%d]", expected)
-	log.Infof("daemonset nodeStatus.Available [%s]", nodeStatus.Available)
+	//log.Infof("daemonset expected [%d]", expected)
+	//log.Infof("daemonset nodeStatus.Available [%s]", nodeStatus.Available)
```

Comment:
@abhi16394 Why did you comment these log statements? They might be useful for debugging.

Reply:
uncommented the log statements